src/pages/ResearcherReview.js

```diff
@@ -51,20 +53,37 @@ class ResearcherReview extends Component {
 
   async findResearcherInfo() {
 
-    let researcher = await Researcher.getPropertiesByResearcherId(this.props.match.params.dacUserId);
+    const user = await User.getById(this.props.match.params.dacUserId);
+    let researcherProps = {
+      academicEmail: user.email,
+      nihUserName: findPropertyValue(UserProperties.NIH_USERNAME, user),
+      linkedIn: findPropertyValue(UserProperties.LINKEDIN, user),
+      orcid: findPropertyValue(UserProperties.ORCID, user),
+      researcherGate: findPropertyValue(UserProperties.RESEARCHER_GATE, user),
+      department: findPropertyValue(UserProperties.DEPARTMENT, user),
+      division: findPropertyValue(UserProperties.DIVISION, user),
+      address1: findPropertyValue(UserProperties.ADDRESS1, user),
+      address2: findPropertyValue(UserProperties.ADDRESS2, user),
+      zipcode: findPropertyValue(UserProperties.ZIPCODE, user),
+      city: findPropertyValue(UserProperties.CITY, user),
+      state: findPropertyValue(UserProperties.STATE, user),
+      country: findPropertyValue(UserProperties.COUNTRY, user),
+      isThePI: findPropertyValue(UserProperties.IS_THE_PI, user),
+      havePI: findPropertyValue(UserProperties.HAVE_PI, user),
+      piName: "",
+      piEmail: "",
+      pubmedID: findPropertyValue(UserProperties.PUBMED_ID, user),
+      scientificURL: findPropertyValue(UserProperties.SCIENTIFIC_URL, user)
+    };
```

Comment:
This is almost the exact same code as we have in `ApplicationDownloadLink` (line 158) with some minor variations. It would be nice to make a call to a utils method that returned all of this content for the user as a single construct.

Reply:
Okay, I just made a helper in utils, so I'm re-requesting review